keyboards/ducky/one2mini/one2mini.c

```diff
@@ -0,0 +1,127 @@
+/* Copyright 2019 /u/KeepItUnder
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "one2mini.h"
+
+/**
+ * @brief   Board-specific bootmagic code.
+ * @details We want to avoid entering bootloader without holding two keys.
+ *          In this case the Ducky originals - "D" and "L".
+ *          These are D {2,3} and L {2,9}
+ *          We also need to avoid debounce by scanning twice with delay.
+ */
+void bootmagic_lite(void) {
+    matrix_scan();
+    wait_ms(DEBOUNCING_DELAY);
```

Comment:
```suggestion
    wait_ms(DEBOUNCE);
```

Or: 
```suggestion
    wait_ms(DEBOUNCE * 2);
```

Reply:
Does this board even need its own bootmagic lite implementation?

Reply:
I don't think so.  It looks pretty stock.